### PR TITLE
DM-35683: Remove references to columns (colKey and columnKey)

### PR DESCRIPTION
**Pull request by fred3m**

### CODE REVIEW

File: python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py [hunk 1]
```diff
@@ -75,15 +75,15 @@ def __call__(self, data: KeyedData, **kwargs) -> Scalar:
...
+    vectorKey = Field[str](doc="Column key to compute scalars")
 
     snFluxType = Field[str](doc="column key for the flux type used in SN selection")
 
     def setDefaults(self):
         super().setDefaults()
-        self.scalarActions.median = MedianAction(colKey=self.columnKey)
-        self.scalarActions.sigmaMad = SigmaMadAction(colKey=self.columnKey)
-        self.scalarActions.count = CountAction(colKey=self.columnKey)
+        self.scalarActions.median = MedianAction(veectorKey=self.vectorKey)
```
Comment (bechtol) [inline python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py:84]: Notice typo `veectorKey`

File: python/lsst/analysis/tools/actions/scalar/scalarActions.py [hunk 2]
```diff
@@ -10,62 +10,62 @@
 
 
 class MedianAction(ScalarAction):
-    colKey = Field[str]("Key of Vector to median")
+    veectorKey = Field[str]("Key of Vector to median")
```
Comment (bechtol) [inline python/lsst/analysis/tools/actions/scalar/scalarActions.py:13]: Notice typo `veectorKey`

File: python/lsst/analysis/tools/analysisMetrics/analysisMetrics.py [hunk 3]
```diff
@@ -280,10 +280,10 @@ def setDefaults(self):
         self.prep.selectors.skyObjectSelector = SkyObjectSelector()
         self.prep.selectors.skyObjectSelector.bands = ["{band}"]
 
-        self.process.calculateActions.medianSky = MedianAction(colKey=f"{{band}}_{self.fluxType}")
-        self.process.calculateActions.meanSky = MeanAction(colKey=f"{{band}}_{self.fluxType}")
-        self.process.calculateActions.stdevSky = StdevAction(colKey=f"{{band}}_{self.fluxType}")
-        self.process.calculateActions.sigmaMADSky = SigmaMadAction(colKey=f"{{band}}_{self.fluxType}")
+        self.process.calculateActions.medianSky = MedianAction(veectorKey=f"{{band}}_{self.fluxType}")
```
Comment (bechtol) [inline python/lsst/analysis/tools/analysisMetrics/analysisMetrics.py:283]: `veectorKey` several instances

File: python/lsst/analysis/tools/analysisParts/shapeSizeFractional.py [hunk 4]
```diff
@@ -36,17 +36,17 @@ def __call__(self, data: KeyedData, **kwargs) -> Scalar:
...
     snFluxType = Field[str](doc="column key for the flux type used in SN selection")
 
     selector = ConfigurableActionField[VectorAction](doc="Selector to use before computing Scalars")
 
     def setDefaults(self):
         super().setDefaults()
-        self.scalarActions.median = MedianAction(colKey=self.columnKey)  # type: ignore
-        self.scalarActions.sigmaMad = SigmaMadAction(colKey=self.columnKey)  # type: ignore
-        self.scalarActions.count = CountAction(colKey=self.columnKey)  # type: ignore
+        self.scalarActions.median = MedianAction(veectorKey=self.vectorKey)  # type: ignore
```
Comment (bechtol) [inline python/lsst/analysis/tools/analysisParts/shapeSizeFractional.py:47]: `veectorKey` several instances

File: tests/test_contexts.py [hunk 5]
```diff
@@ -73,13 +73,13 @@ def getInputSchema(self) -> KeyedDataSchema:
         return (("a", Vector),)
```
Comment (bechtol) [inline tests/test_contexts.py:73]: `veectorKey` several instances

**Review by bechtol — APPROVED** [review]
I have searches all the files for any instances of `columnKey` and `colKey` and do not find any instances.

There are multiple files that have `veectorKey` typo. Looks good once the typo is fixed.

File: python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py (hunk 1 above)
Comment (fred3m) [reply python/lsst/analysis/tools/actions/plot/scatterplotWithTwoHists.py]: 🤦‍♂️

**Pull request merged**